### ISSUE TITLE
manually exposing third party modules form the base composer.json file is a little tricky.

### ISSUE THREAD

**Issue by sasky** [silverstripe/vendor-plugin#17]
Hello. Just a small issue. 
I was trying to manually expose third party files like this in my base composer.json file.
```
    "extra": {
        "expose": ["module/images", 
                         "another-module/javascript"]

```

But for some reason they were not being recognised.  So after pulling out my remaining hair. I started tinkering with the vendor-plugin source code. And discovered that the composer.json file is not recognised it it does not have 
```
  "type": "silverstripe-something",
```
in the composer.json file. 
That comes from the getExposedFolders() in Library.php line 230.
```
      // Only expose if correct type
        if (empty($data['type']) || !preg_match(VendorPlugin::MODULE_FILTER, $data['type'])) {
            return [];
        }
```

Maybe get rid of that restriction for the base composer.json file? ( I can do a pull request if you like) 
Or at least mention that is needed in the docs?

Cheers
Cam

**Comment by chillu** [comment]
Yeah I don't see a reason that you'd need a composer `type` on your project's own `composer.json` - that's a bug.

**Comment by bergice** [comment]
Looks like this was already resolved with https://github.com/silverstripe/vendor-plugin/pull/18/commits/cb165e1d39f11e159915cf0b3bfccd752cefce48

Could you upgrade your SilverStripe CMS version and see if that resolves your issue?
